include/vapor/OSPRay.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#ifdef BUILD_OSPRAY
+#include <ospray/ospray_util.h>
+#endif
+#include <string>
+#include <glm/glm.hpp>
```

Comment:
This header, OSPray.h, is only included by VAPOR if build ospray is enabled, is it right? 
If so, what's the point of having some code that for situations where BUILD_OSPRAY isn't defined?

Reply:
Yes but I don’t understand the question.

Reply:
Will a situation ever occur where this particular header file is included, but the `BUILD_OSPRAY` macro is NOT defined? Line 2 seems to be testing that particular situation.

Reply:
No? I still don’t follow.

Reply:
If the inclusion of this header and the definition of macro BUILD_OSPRAY always happen or not happen at the same time, then removing line 2 and line 4 will not change any logic, is it right?

Reply:
Oh. No vapor/Ospray.h is still included when BUILD_OSPRAY is not set